backend/typescript/rest/campRoutes.ts

```diff
@@ -26,9 +27,10 @@ const fileStorageService: IFileStorageService = new FileStorageService(
 
 const campService: ICampService = new CampService(fileStorageService);
 /* Get all camps */
-campRouter.get("/", async (req, res) => {
+campRouter.get("/", getCampDtoValidator, async (req, res) => {
   try {
-    const camps = await campService.getCamps();
+    const { campYear } = req.query;
```

Comment:
If the URL doesn't include a year, should we default it to the current year? I can't imagine the frontend needing all the camps across all the years. The default view for the camp list should be the current year too

Reply:
Hmm I think it makes sense to include all the camps when no year is specified, even if we don't currently have a year specified. For a GET "all" endpoint, I think it should behave that way be default unless an explicit parameter is included. For when we want the current year's camps, our frontend should have to explicitly say so

Reply:
ok fair enough. I am good with it then